### ISSUE TITLE
no-restricted-properties fails when specifying more than one restricted property

### ISSUE THREAD

**Issue by adamterlson** [eslint/eslint#7167]
* **ESLint Version:** 3.5.0
* **Node Version:** 4.1.2
* **npm Version:** 3.5.3

**What parser (default, Babel-ESLint, etc.) are you using?** default

In my eslintrc I've included no-restricted-properties and copy-pasted from [the example](http://eslint.org/docs/rules/no-restricted-properties).

```yml
rules:
    no-restricted-properties: [2, [{
            "object": "disallowedObjectName",
            "property": "disallowedPropertyName"
        }, {
            "object": "disallowedObjectName",
            "property": "anotherDisallowedPropertyName",
            "message": "Please use allowedObjectName.allowedPropertyName."
        }]]
```

Running eslint results in the following error:

```
/Users/.../.eslintrc.yml:
       	Configuration for rule "no-restricted-properties" is invalid:
       	Value "[object Object],[object Object]" is the wrong type.
```

If I replace the array of objects with a singular object, the validation passes and the rule works just fine.

**Comment by mysticatea** [comment]
Thank you for this issue.

Yes, the document is wrong currently. It has fixed in https://github.com/eslint/eslint/pull/7099/files

**Comment by platinumazure** [comment]
The documentation will be fixed with our next release- in the meantime, you just need to remove the extra array brackets.

Closing as this seems to be resolved- please reopen if you disagree.

**Comment by adamterlson** [comment]
Yup, that worked!  Thanks.
